nx-X11/programs/Xserver/mi/mibank.c

```diff
@@ -175,8 +175,8 @@ typedef struct _miBankQueue
         (*pScreenPriv->BankInfo.SetDestinationBank)(pScreen, (_no)) - \
         (pScreenPriv->BankInfo.BankSize * (_no)))
 
-#define ALLOCATE_LOCAL_ARRAY(atype, ntype) \
-    (atype *)ALLOCATE_LOCAL((ntype) * sizeof(atype))
+#define malloc_ARRAY(atype, ntype) \
+    (atype *)malloc((ntype) * sizeof(atype))
```

Comment:
Yes, it is an artifact but it is like upstream handels that. Once mibank will be gone the will vanish anyway. So maybe cherrypicking Ionic's backport of dropping mibank should be done now.